x-pack/legacy/plugins/siem/public/pages/case/components/case_view/index.tsx

```diff
@@ -293,14 +217,9 @@ export const CaseComponent = React.memo<CaseProps>(({ caseId, initialData, isLoa
               />
               <TagList
                 data-test-subj="case-view-tag-list"
-                tags={tags}
-                iconAction={{
-                  'aria-label': title,
-                  iconType: 'pencil',
-                  onSubmit: newTags => onUpdateField('tags', newTags),
-                  onClick: isEdit => setIsEditTags(isEdit),
-                }}
-                isEditTags={isEditTags}
+                tags={data.tags}
+                onSubmit={newTags => onUpdateField('tags', newTags)}
```

Comment:
nit to avoid lambda function or you can do a `useCallback`
```suggestion
                onSubmit={onUpdateField.bind(null, 'tags')}
```